backend/db/migrations/20260717120000_add_loopback.up.sql

```
@@ -0,0 +1,90 @@
+    session_id       UUID NOT NULL REFERENCES loopback_sessions(id) ON DELETE CASCADE,
+    job_execution_id UUID NOT NULL REFERENCES job_executions(id) ON DELETE CASCADE,
+    round            INTEGER NOT NULL,
+    role             TEXT NOT NULL CHECK (role IN ('original', 'rerun')),
+    is_mutatable     BOOLEAN NOT NULL DEFAULT FALSE,
+    origin_job_id    UUID REFERENCES job_executions(id) ON DELETE SET NULL,
+    created_at       TIMESTAMPTZ NOT NULL DEFAULT NOW(),
+    UNIQUE (session_id, job_execution_id)
+);
+
+CREATE INDEX IF NOT EXISTS idx_loopback_session_jobs_session
+    ON loopback_session_jobs(session_id);
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)